src/components/Sections/SectionList/index.jsx

```diff
@@ -0,0 +1,164 @@
+import React from 'react';
+
+import { format, formatDistance, formatRelative, subDays } from 'date-fns';
+
+import './index.scss';
+
+const SectionList = ({
+  name,
+  description,
+  keywords,
+  url,
+  roles,
+  entity,
+  type,
+  startDate,
+  endDate,
+  highlights,
+  position,
+  summary,
+  organization,
+  sectionName,
+  reference,
+  language,
+  fluency,
+  institution,
+  studyType,
+  area,
+  score,
+  courses,
+  title,
+  date,
+  awarder,
+  publisher,
+  releaseDate,
+  level,
+}) => {
+  const injectHTML = (content, key) => {
```

Comment:
Optimization tip: Either we can move out this function outside this component or useCallback. Although for smaller projects it will not give issue. But its good to know about these practices.